faucet/faucet.py

```diff
@@ -94,6 +95,7 @@ class Faucet(RyuAppBase):
         EventFaucetFastStateExpire: ('fast_state_expire', 2),
         EventFaucetAdvertise: ('advertise', 15),
         EventFaucetFastAdvertise: ('fast_advertise', 5),
+        EventFaucetEventSockHeartbeat: ('event_sock_heartbeat', 300),
```

Comment:
seems super long -- why can't we just do 10 or something?  It's not like it's an expensive operation, and I think will make the restart times tolerable!

Reply:
Event queue is only 120 long. Don’t want it to overflow. Though I guess since we push state on connection, it should be fine. But I still don’t  want to be as short as 10s. Thoughts, @grafnu ?